src/ssl.c

```diff
@@ -43691,18 +43707,10 @@ void* wolfSSL_GetHKDFExtractCtx(WOLFSSL* ssl)
         return wolfSSL_X509_sign(x509, ctx->pctx->pkey, wolfSSL_EVP_MD_CTX_md(ctx));
     }
 #endif /* OPENSSL_EXTRA */
+#endif /* WOLFSSL_CERT_GEN */
 
-/* Guarded by either
- * A) WOLFSSL_WPAS_SMALL is on or
- * B) (OPENSSL_EXTRA or OPENSSL_EXTRA_X509_SMALL) + WOLFSSL_CERT_GEN +
- *    (WOLFSSL_CERT_REQ or WOLFSSL_CERT_EXT or OPENSSL_EXTRA) has been
- *    defined
- */
-#if defined(WOLFSSL_WPAS_SMALL) || \
-    (defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)) && \
-    defined(WOLFSSL_CERT_GEN) && \
-    (defined(WOLFSSL_CERT_REQ) || defined(WOLFSSL_CERT_EXT) || \
-     defined(OPENSSL_EXTRA))
+
+#if defined(OPENSSL_ALL) || defined(OPENSSL_EXTRA)
```

Comment:
This is also needed for `--enable-opensslextra=x509small` `OPENSSL_EXTRA_X509_SMALL`.

Reply:
Done!